packages/logger/src/index.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { assert } from 'chai'; // tslint:disable-line:no-implicit-dependencies
```

Comment:
this PR also pushes for tests to be in TypeScript, and [we want to move that decision](https://github.com/slackapi/node-slack-sdk/pull/808#issuecomment-530593159) outside unrelated PRs.